horovod/spark/keras/remote.py

```diff
@@ -154,7 +157,14 @@ def train(serialized_model, train_rows, val_rows, avg_row_size):
                 if _checkpoint_callback:
                     _checkpoint_callback.filepath = ckpt_file
                 else:
-                    _checkpoint_callback = k.callbacks.ModelCheckpoint(ckpt_file)
+                    if is_dbfs and LooseVersion(tf.__version__) < LooseVersion("2.0.0"):
```

Comment:
Similar to previous comment: we should be able to adopt this pattern universally after we drop BareKeras, without needing to special-case DBFS.